controller/user_controller.py

```diff
@@ -0,0 +1,75 @@
+import os
+import psycopg2
+
+from model.user_model import User
+
+
+class UserController:
+    session = None
+    def __init__(self, session):
+        self.session = session
+
+
+    def add_user(self, user_data):
+        self.session.add(user_data)
+        try:
+            self.session.commit()
+        except:
+            self.session.rollback()
+
+    def delete_user(self, id):
+        self.session.query(User).filter_by(id=id).delete()
+        try:
+            self.session.commit()
+        except:
+            self.session.rollback()
+
+    def get_user(self, id):
+        user = self.session.query(User).filter_by(id=id).first()
+        return user
+
+
+    def update_user(self, user_data):
```

Comment:
thia function doesn't do anything?

Reply:
In the code, I retrieve a row, then I each field in the row. 

Like I mention, these changes won't be made in the database without committing. 

That was why I sent it here so those changes can be committed. 

Don't know if it makes sense.